docs/_sidebar.md

```diff
@@ -4,14 +4,12 @@
 
 * **Getting Started**
 
-
   * [Interlay 101](getting-started/interlay-101.md)
   * [Learn interBTC](getting-started/interbtc.md)
   * [Governance](getting-started/governance.md)
-  * [FAQ :arrow_upper_right:](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
-
+  * [FAQ 🡥](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
```

Comment:
```suggestion
  * [FAQ :bookmark_tabs:](https://interlay.notion.site/FAQ-Page-5e3019b1cfd94f6693dc186e9640e607)
```

Reply:
I used the error to indicate an external link. Not sure what the emoji should do?